Kconfig.zephyr

```diff
@@ -184,6 +184,12 @@ endmenu
 
 menu "Compiler Options"
 
+config ZEPHYR_SYSTEM_INTERFACE
+	bool "Configure Zephyr include paths as system headers"
+	help
+	  Marks Zephyrs include paths as system headers. Preventing warnings during compilation of a user application in case stricter warning levels are configured.
+	  option is set.
```

Comment:
this `option is set.` looks like some leftover :) let's remove it and perhaps add `If unsure, leave at the default value 0.` similarly to other options